### DISCUSSION TITLE
Add default value and possible values/value type to documentation

### DISCUSSION THREAD

**Discussion by SenseiMarv** [ghostty-org/ghostty#3872] [Feature Requests, Ideas] [answered]
When reading the documentation, it is difficult for me to decipher what the default value of a setting is and what values are allowed in Ghostty. While there is often a text description where the default is stated and there is a list showing what values are allowed, e.g. here: https://ghostty.org/docs/config/reference#cursor-style-blink, this is not always clearly stated and makes it difficult to find information quickly by having to parse large chunks of text, e.g. here there is no clear statement of what the default is: https://ghostty.org/docs/config/reference#cursor-style.

I'd like to discuss whether it's possible to introduce a clear short statement of what the default is and what values are allowed/what type the value is. This would probably only affect the [Option Reference](https://ghostty.org/docs/config/reference) page (can be discussed if someone sees a value for adding this to other pages as well).

Examples of what I have in mind:

Example 1:
# [`font-family`](https://ghostty.org/docs/config/reference#font-family)

Value: `string`

The font families to...

---

Example 2:
# [`font-size`](https://ghostty.org/docs/config/reference#font-size)

Value: `number` (Default: `14`) <- Unclear what even is the default here, as there currently is no clear statement.

Font size in points. This value...

---

Example 3:
# [`cursor-style`](https://ghostty.org/docs/config/reference#cursor-style)

Value:
  - `block` (Default)
  - `bar`
  - `underline`
  - `block_hollow`

The style of the cursor. This sets...

**Comment by JakeWharton** [comment, marked as answer]
Same as #3568, although this has more detail.

**Reply by SenseiMarv** [reply]
Thanks for the hint, I didn't find the linked discussion before. Should I add my comment there and close this or maybe ask to close the other in favor of this more detailed explanation - any preference?

**Reply by JakeWharton** [reply]
Closing, and adding a comment there sounds nice to me.
